src/data_sync/common.py

```diff
@@ -1,13 +1,72 @@
 """Shared methods between both sync scripts."""
 
 from datetime import datetime, timezone
-from typing import List, Tuple
+
+from dateutil.relativedelta import (
+    relativedelta,
+)  # dateutil is currently not explicitly required in requirement.in, only installed via dune
```

Comment:
If you're gonna import it directly, it should be added to the requirements.

Also,

```suggestion
from dateutil.relativedelta import relativedelta
```

Reply:
Yes, I know. I am just scared to add it to the requirements.in because recompiling would change a lot of dependencies.

Let me try though, and see what breaks.

Reply:
Good reason to keep dependencies updated... 

You could also just install a fixed version of dateutil (i.e. the one that dune uses).

Reply:
Adding it to `requirements.in` cased less issues than expected. Probably some of the fixed dependencies keep us on some old version. Will look into that separately.